src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -4900,8 +4901,11 @@ public LinkedHashMap<InetAddress, Float> effectiveOwnership(String keyspace) thr
         else
         {
             List<String> userKeyspaces = Schema.instance.getUserKeyspaces();
-
-            if (userKeyspaces.size() > 0)
+            if (Schema.instance.getKeyspaceInstance(SystemPalantir.NAME) != null)
+            {
+                keyspace = SystemPalantir.NAME;
+            }
```

Comment:
Instead of modifying it here (I feel it might be confusing if we use this functions in other pieces of the code), wanna add to  the`Status` class instead: `ownerships = probe.effectiveOwnership(keyspace != null ? keyspace: SystemPalantir.NAME);` and add a small comment on why we are doing this.

Reply:
I think this is the correct place for this logic because this method already contains logic to auto resolve a keyspace if none is provided

Reply:
Also, adding the system_palantir keyspace name as the argument to `effectiveOwnership` isn't great because then the method would throw in the case that system_palantir is not present, whereas in my patch it would instead fall back to the existing behavior.